tests/components/samsungtv/test_config_flow.py

```diff
@@ -566,12 +526,17 @@ async def test_ssdp_legacy_not_remote_control_receiver_udn(
 
 @pytest.mark.usefixtures("remote", "rest_api_failing")
 async def test_ssdp_noprefix(hass: HomeAssistant) -> None:
-    """Test starting a flow from discovery without prefixes."""
+    """Test starting a flow from discovery when friendly name doesn't start with [TV]."""
+    ssdp_data = deepcopy(MOCK_SSDP_DATA)
+    ssdp_data.upnp[ATTR_UPNP_FRIENDLY_NAME] = ssdp_data.upnp[ATTR_UPNP_FRIENDLY_NAME][
+        4:
+    ]
```

Comment:
Same here, since we don't have a "real-life" fixture, I think it is clearer to re-use the default fixture and tweak inside the test